src/lib/dom.generated.d.ts

```diff
@@ -4327,59 +4444,59 @@ interface DocumentEventMap extends GlobalEventHandlersEventMap, DocumentAndEleme
     "fullscreenerror": Event;
     "pointerlockchange": Event;
     "pointerlockerror": Event;
-    "readystatechange": ProgressEvent;
+    "readystatechange": ProgressEvent<Document>;
     "visibilitychange": Event;
 }
 
 /** Any web page loaded in the browser and serves as an entry point into the web page's content, which is the DOM tree. */
 interface Document extends Node, NonElementParentNode, DocumentOrShadowRoot, ParentNode, XPathEvaluatorBase, GlobalEventHandlers, DocumentAndElementEventHandlers {
-    /**
-     * Sets or gets the URL for the current document.
+    /**
+     * Sets or gets the URL for the current document.
      */
     readonly URL: string;
-    /**
-     * Gets the object that has the focus when the parent document has focus.
+    /**
+     * Gets the object that has the focus when the parent document has focus.
      */
     readonly activeElement: Element | null;
-    /**
-     * Sets or gets the color of all active links in the document.
+    /**
+     * Sets or gets the color of all active links in the document.
      */
     /** @deprecated */
     alinkColor: string;
-    /**
-     * Returns a reference to the collection of elements contained by the object.
+    /**
+     * Returns a reference to the collection of elements contained by the object.
      */
     /** @deprecated */
     readonly all: HTMLAllCollection;
-    /**
-     * Retrieves a collection of all a objects that have a name and/or id property. Objects in this collection are in HTML source order.
+    /**
+     * Retrieves a collection of all a objects that have a name and/or id property. Objects in this collection are in HTML source order.
      */
     /** @deprecated */
     readonly anchors: HTMLCollectionOf<HTMLAnchorElement>;
-    /**
-     * Retrieves a collection of all applet objects in the document.
+    /**
+     * Retrieves a collection of all applet objects in the document.
      */
     /** @deprecated */
     readonly applets: HTMLCollectionOf<HTMLAppletElement>;
-    /**
-     * Deprecated. Sets or retrieves a value that indicates the background color behind the object.
+    /**
+     * Deprecated. Sets or retrieves a value that indicates the background color behind the object.
      */
     /** @deprecated */
     bgColor: string;
-    /**
-     * Specifies the beginning and end of the document body.
+    /**
+     * Specifies the beginning and end of the document body.

```

Comment:
👆 what is all this, LF/CRLF variation?

Looks set to create trouble down the line.

Reply:
Looks like comments.json (1) still has explicit `\r\n` and (2) something changed recently to stop converting them to `\n`.

The right fix is to explicitly change comments.json to use `\n`.

Reply:
Good idea!

People may have weird git configs, check in and then CRLF noise could obscure real changes for PR reviews.